src/chrome/komodo/content/keybindings/keybindings.p.xml

```diff
@@ -608,32 +577,14 @@
                     <![CDATA[
                     try {
                         var numlabels = this.keylabels.length;
-                        var usePopup = numlabels>1;
                         if (numlabels < 2) {
                             if (numlabels == 0) {
-                                this.currentkey.value = this._bundle.GetStringFromName("none.message");
-                                this.currentlabel = '';
+                                this.currentkey.value = "None"; // this._bundle.GetStringFromName("none.message");
```

Comment:
For some reason, `this._bundle` was undefined, so I've replaced it with the "None" word.